app/services/call_creator.rb

```diff
@@ -10,9 +11,7 @@ def run
                      member_id: @params[:member_id],
                      member_phone_number: @params[:member_phone_number],
                      target_index: @params[:target_index])
-    if @call.save
-      place_call
-    end
+    place_call if @call.save
```

Comment:
@osahyoun is this because of rubocop? I'm not a fan of this format. When the guard condition is not just checking a condition, but it's actually doing important stuff (saving the model!), I rather have it be in the explicit if condition. Not a big deal tho :P, so no need to change it.